Sources/Bootstrap/Tags/Alert.swift

```diff
@@ -1,20 +1,11 @@
 import Leaf
 import TemplateKit
 
+/// Bootstrap Alert Tag
 public final class AlertTag: TagRenderer {
-    public enum Keys: String {
-        case primary = "primary"
-        case secondary = "secondary"
-        case success = "success"
-        case danger = "danger"
-        case warning = "warning"
-        case info = "info"
-        case light = "light"
-        case dark = "dark"
-    }
 
     public func render(tag: TagContext) throws -> EventLoopFuture<TemplateData> {
```

Comment:
I'm a bit picky about this but could you substitute `EventLoopFuture` with `Future` throughout?

Reply:
Sure.. what is fun is that Future is in the Protocol.. But if you let Xcode fix it goes to EventLoopFuture